packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -275,6 +285,26 @@ class PouchLink extends CozyLink {
       return this.executeQuery(operation)
     }
   }
+  /**
+   *
+   * @param {string} doctype
+   * @returns {boolean} the need to wait for the warmup
+   * Check if there is warmup queries for this doctype
+   * and return if those queries are already warmedup or not
```

Comment:
Please put the description the docstring above the definition of the params, with a line break between params and description.